tests/app/views/test_marketplace.py

```diff
@@ -469,6 +469,24 @@ def test_supplier_applied_view_application(self):
 
         self._assert_view_application(document, brief_id)
 
+    def test_supplier_applied_view_application_for_closed_opportunity(self):
+        self.login_as_supplier()
+        # mocking that we have applied
+        self._data_api_client.find_brief_responses.return_value = {
+            "briefResponses": [{"lazy": "mock"}],
+        }
+        brief = self.brief.copy()
+        brief['briefs']['status'] = "closed"
+        brief['briefs']['publishedAt'] = "2000-01-25T12:00:00.000000Z"
```

Comment:
Yep that was some lazy copy-pasting from a different test for closed briefs.  Have removed them now.